lib/msal-core/build/npm-build-steps.yml

```diff
@@ -7,11 +7,12 @@ steps:
   displayName: 'Install Node.js'
 
 - task: Npm@1
+  env:
+    NPM_CONFIG_TOKEN: $(NPM_CONFIG_TOKEN)
```

Comment:
Whose token is this ?

Reply:
I generated one from my account to get it working.

Reply:
can we get an organizational one to use so that its not tied to an individual ?

Reply:
We could, but that would make managing it harder (for essentially no security benefit, given its currently just for reading packages).

Reply:
I think its worth looking into just to make sure we don;t have things break if an individual member moves teams or something